resources/test/fixtures/N804.py

```diff
@@ -30,6 +30,10 @@ def __new__(cls, *args, **kwargs):
     def __init_subclass__(self, default_name, **kwargs):
         ...
 
+    @classmethod
+    def class_method_with_positional_only_argument(cls, x, /, other):
+        ...
```

Comment:
(Should we add one variant where `cls` _isn't_ the first argument, but it does have positional-only args?)